### PR TITLE
Adding remote reminders

### PR DESCRIPTION
**Pull request by doublerainbo**

### CODE REVIEW

**Comment by doublerainbo** [comment]
🚀

**Comment by doublerainbo** [comment]
rofl too bad rocket doesn't work here

File: src/Pill.ino [hunk 1]
```diff
@@ -62,6 +94,13 @@ void loop() {
   int currentSecond = Time.second();
   if (currentSecond != lastActiveSecond) {
     lastActiveSecond = currentSecond;
+    if (blinkLed) {
+      if (ledIsOn) {
+        ledOff();
+      } else {
+        ledOn();
```
Comment (xeniatay) [inline src/Pill.ino:101]: eh? it'll only turn on when blinked?

Comment (doublerainbo) [reply src/Pill.ino:101]: the night time event is what turns it on, this is only if blinking is active

**Review by xeniatay — APPROVED** [review]

**Comment by xeniatay** [comment]
:shipit: i guess 😂

**Comment by doublerainbo** [comment]
🎰

**Pull request merged**